File: Session_2/Bài 1.4/Bài 1.4.2 Sum 1 to n/Sum.jl
# đây là chương trình julia về tính tổng 1 đến n
function sum_1_to_n(n)
    s = 0 
    for i in 1:n
        s += i
    end
    return s
end

println("Nhap n: ")
n = parse(Int64, readline())
result = sum_1_to_n(n)
println("Tong tu 1 den n la: ", result)

File: Session_2/Bài 1.4/Bài 1.4.2 Sum 1 to n/sum.cpp
// Sum 1 to n
#include <iostream>
using namespace std;

//hàm tính tổng từ 1 đến n
int sum_1_to_n(int n)
{
    int s = 0;
    for (int i = 1; i <= n; i++)
    {
        s += i;
    }
    return s;
}
int main()
{

    cout << " nhap n: ";
    int n;
    cin >> n;
    //gọi hàm sum_1_to_n và in kết quả
    int result = sum_1_to_n(n);
    cout << "Sum = " << result << endl;
    return 0;
}